src/bootstrap/config.rs

```diff
@@ -785,12 +783,25 @@ impl Config {
             config.llvm_ldflags = llvm.ldflags.clone();
             set(&mut config.llvm_use_libcxx, llvm.use_libcxx);
             config.llvm_use_linker = llvm.use_linker.clone();
-            config.llvm_allow_old_toolchain = llvm.allow_old_toolchain;
-            config.llvm_polly = llvm.polly;
+            config.llvm_allow_old_toolchain = llvm.allow_old_toolchain.unwrap_or(false);
+            config.llvm_polly = llvm.polly.unwrap_or(false);
             config.llvm_from_ci = match llvm.download_ci_llvm {
                 Some(StringOrBool::String(s)) => {
                     assert!(s == "if-available", "unknown option `{}` for download-ci-llvm", s);
-                    config.build.triple == "x86_64-unknown-linux-gnu"
+                    // FIXME: this is duplicated in bootstrap.py
+                    // This is currently all tier 1 targets (since others may not have CI artifacts)
+                    // https://doc.rust-lang.org/rustc/platform-support.html#tier-1
```

Comment:
does the fixme include the rest of the comment lines or could it go below them? i.e:

```suggestion
                    // This is currently all tier 1 targets (since others may not have CI artifacts)
                    // https://doc.rust-lang.org/rustc/platform-support.html#tier-1
                    // FIXME: this is duplicated in bootstrap.py
```

Reply:
It could go either way I think. Ideally it would be handled in only one place and the comment wouldn't be necessary at all. Do you think moving the FIXME makes it easier to read.

Reply:
slightly, in that you don't have to work out if "this is currently all tier 1 targets..." is meant to be fixed somehow